export.sh

```diff
@@ -11,8 +11,12 @@ realpath_int() {
     echo "$scriptdir"
 }
 
+__verbose() {
+    [[ -n ${IDF_QUIET} ]] && return
```

Comment:
Sorry, i just ran shellcheck and realized that this `[[` won't work in `dash`. Can fix this up while rebasing on the other PR i've mentioned.

Reply:
One more thing, how do you feel about using IDF_EXPORT_QUIET or IDF_TOOLS_QUIET instead of IDF_QUIET? I'm afraid that IDF_QUIET would imply to users that *all* the tools would disable output. Meeting that expectation would require quite a bit more changes to be added.

Reply:
@igrr please take it from here and do as you see fit.  Just some suggestions.  Thanks for the portability check.

Reply:
@igrr are you going to assume responsibility for this?

Reply:
Sorry, have missed your previous comment. Sure, can take it from here. Thank you.